randovania/games/fusion/gui/ui_files/games_tab_fusion_widget.ui

```diff
@@ -136,17 +136,54 @@
+           <string>Randovania and the MARS patcher make some changes to the original game in order to improve the game experience or to simply fix bugs in the original game. Many changes are optional and can be disabled in the options Randovania provides, but the following are always there: 
+
+### Gameplay
+
+- The Security Rooms are now item locations and the locks can be shuffled into the item pool. 
+
+- Screw Attack will no longer allow Single Wall Jumps, Wall Jump behaviour is now the same as without Screw Attack. 
+
+- Hints have been placed at Navigation Stations. More details found on the Hints tab.
+
+- Sector 6 Varia Core-X always spawns regardless of having Charge Beam or not. 
+
+- SA-X no longer requires Plasma Beam to be defeated. 
+
+### Room Changes
+
+- Main Deck Operations Room - is now a L0 Grey Door, this means the Red Keycard is not hard required to beat the game.
+
+- Main Deck Operations Ventillation - now has a connection between Crew Quarters East/West.
+
+- Main Deck Central Reactor Core - now has a platform near the top to make removing the vines not dangerous.
+
+- Sector 5 will always remain in it's unbroken state, some rooms have been modified to account for this. 
```

Comment:
```suggestion
- The destruction of Sector 5 has been changed to be always consistent. Everything in West Sector 5 will always be repaired, while everything East Sector 5 will be destroyed.
```
Alternatively instead of east/west, you can use arctic containment or crows nest instead.